docs/keto/guides/userset-rewrites.mdx

```diff
@@ -1,12 +1,11 @@
-id: userset-rewrites title: Define file access permissions with userset rewrites sidebar_label: Userset rewrites
-
+---
+id: userset-rewrites
+title: Define global permission rules using OPL
+sidebar_label: Global Rules
```

Comment:
```suggestion
sidebar_label: Global rules
```

nit: For document titles and in-document headings we agreed to use sentence case instead of title case.